### DISCUSSION TITLE
How to Restrict Dropping Sections inside another Section ?

### DISCUSSION THREAD

**Discussion by AgnRakul** [GrapesJS/grapesjs#4729] [General]
The Requirement is to Restrict Dropping Sections inside another Section but it should drop in down or up this example is avaible in grapesjs mjml demo and one more qns this concept is possible only using mjml ? or we can do it with default grapesjs Options

**Comment by ronaldohoch** [comment]
Hello!
We have an html attribute `data-gjs-draggable`:

```js
      editor.BlockManager.add('columns', {
        label: "Column",
        category: "Layouts",
        content: `<table data-gjs-droppable="["tbody", "thead"]" data-gjs-draggable="[data-gjs-type=wrapper]" class="table-row">
          <tr>
            <td class="table-cell">
              <p>Text</p>
            </td>
          </tr>
          </table>`,
      });
```

`data-gjs-draggable="[data-gjs-type=wrapper]"`:
You can put which component it can be draggable in it, wrapper is the body component in canva.
